src/components/state-details.js

```diff
@@ -581,6 +578,23 @@ export default function StateDetailsPage ({ location, data }) {
                 </>
               )}
 
+              <div className="card">
+                <div className="card-body">
+                  <h5 class="card-title">
+                    <a href="https://bit.ly/dirty-power-plants-usa" target="_blank" rel="noreferrer">
+                      See a map of dirty power plants in the US
+                    </a>
+                  </h5>
+                  <a href="https://bit.ly/dirty-power-plants-usa" target="_blank" rel="noreferrer">
```

Comment:
This link is a duplicate of the previous one and should thus not be tabbable. Add `tabindex="-1"`.